vision/vision_service.proto

```diff
@@ -19,6 +21,14 @@ message VisionRequest {
   // Recognitions to perform on the image. The recognitions should be done
   // after the detections.
   repeated EnumFacialRecognitionTag recognition = 3;
+
+  // It is sometimes more convenient to send many images to be processed
+  // by vision services
+  ImageList image_list = 4;
+
+  // or video data if splitting into frames is not convenient
+  // in the client.
```

Comment:
There is a possibility: That you get 1000 images at 10fps an then encode to save bandwidth.  It's better not to anticipate how the messages will be used.